packages/components-react/src/button/Button.tsx

```diff
@@ -25,7 +25,7 @@ const ICON_POSITIONS: { [key: string]: string } = {
 export const LuxButton = (props: LuxButtonProps) => {
   const { size, icon: iconNode, iconPosition, ...otherProps } = props;
 
-  const className = `lux-button ${size !== undefined ? SIZE_CLASSNAME[size] : ''}`;
+  const className = `lux-button ${size !== undefined ? SIZE_CLASSNAME[size] : ''} ${otherProps.className || ''}`;
```

Comment:
Ik kwam er achter dat er geen extra className aan de button kan worden meegegeven, is het goed dat ik dat de fix in deze PR zit? Of kan ik beter een losse PR hiervoor aanmaken?

Reply:
Je hebt het nu hier opgelost, imho: laat het maar in deze PR